src/app/new-auth/new-can-reset-password.guard.ts

```diff
@@ -0,0 +1,21 @@
+import { Injectable } from '@angular/core';
+import { CanActivate, ActivatedRouteSnapshot, RouterStateSnapshot } from '@angular/router';
+import { Observable } from 'rxjs';
+import { Router } from '@angular/router';
+
+@Injectable()
+export class NewCanResetPasswordGuard implements CanActivate {
```

Comment:
Why is there a guard for reseting your password?
Also this guard belongs in 'src/app/core'

Reply:
i think this was auto generated, not sure how it happened, I'll remove it